service/windns/windns.go

```diff
@@ -238,3 +238,52 @@ ForEach ($ns in $toRemove){
 		log.Errorf("ERROR removing nrpt rules: %v", err)
 	}
 }
+
+func IsNrptPoliciesEffective() bool {
+	script := fmt.Sprintf(`Add-DnsClientNrptRule -Namespace ".ziti.test" -NameServers "100.64.0.1" -Comment "Added by ziti-tunnel" -DisplayName "ziti-tunnel:.ziti.test"
```

Comment:
hardcoded nameserver 100.64.0.1 - pass the dns ip to this function or just reference. since the dns server sits on the TUN ip - pass the TUN ip to this function like `windns.AddNrptRules(sc.HostnamesToAdd, rts.state.TunIpv4)`

Reply:
i think it would make sense to use the same probe the monitor service does: `dew-dns-probe.openziti.org`

Reply:
well - none of this matters i guess lol because the last thing the script does is remove all this so.. :)